containers/proxy-httpd-image/uyuni-configure.py

```diff
@@ -124,17 +124,58 @@ def getIPs(fqdn: str) -> Tuple[str, str]:
         if len(proxyIPv6) > 0:
            requireIPv6 = "Require ip {}".format(proxyIPv6)
         file.write(f'''<Directory "/srv/www/tftpsync">
-    <IfVersion >= 2.4>
-        <RequireAny>
-            {requireIPv4}
-            {requireIPv6}
-        </RequireAny>
-    </IfVersion>
+    <RequireAny>
+        {requireIPv4}
+        {requireIPv6}
+    </RequireAny>
 </Directory>
 
 WSGIScriptAlias /tftpsync/add /srv/www/tftpsync/add
 WSGIScriptAlias /tftpsync/delete /srv/www/tftpsync/delete''')
 
+    with open("/etc/apache2/conf.d/saltboot.conf", "w") as file:
+        # Saltboot uses the same URL regardles containerized or normal proxy
+        # here we rewrite URL so upstream server understands it
+
+        # First condition/rule removes double arch rule for backward compatility
+        # second changes /saltboot/image/..?orgid=<org> to /os-images/<org>/
+        file.write('''
+RewriteEngine on
+RewriteCond %{REQUEST_URI} ^/saltboot/image.*x86_64.*x86_64
+RewriteRule ^(.*)\.x86_64(.*\.x86_64-.*) $1$2 [N]
+
+RewriteCond %{QUERY_STRING} orgid=(\d+)
+RewriteRule "^/saltboot/image(.+)$" "/os-images/%1$1"  [R,L,QSD]
+''')
+
+    with open("/etc/apache2/vhosts.d/ssl.conf", "w") as file:
+        file.write(f'''
+<IfDefine SSL>
+<IfDefine !NOSSL>
+<VirtualHost _default_:443>
+
+	DocumentRoot "/srv/www/htdocs"
+	ServerName {config['proxy_fqdn']}
+
+	ErrorLog /proc/self/fd/2
+	TransferLog /proc/self/fd/1
+	CustomLog /proc/self/fd/1   ssl_combined
+
+	SSLEngine on
+	SSLUseStapling  off
```

Comment:
What is the reason for turning this off?

Reply:
I might be too hasty with this here. We do not support it with our generated certificates, so I though about disabling it to get rid of warning messages. But in case customer uses their own certificates, then this can be handy.

I will reenable it.